documentation/cobigen-core_configuration.asciidoc

```diff
@@ -335,6 +335,20 @@ folder
 Let the `cobigen.properties` file contain the line `relocate=../sub2/${cwd}`. Given that, the relative destination path of `Template.java.ftl` will be resolved to `folder/sub2/Template.java`. Compare `xref:templatescan-node[template scan]` configuration for more information about basic path resolution. The `relocate` property specifies a relative path from the location of the `cobigen.properties`. The `${cwd}` placeholder will contain the remaining relative path from the `cobigen.properties` location to the template file. In this basic example it just contains `Template.java.ftl`, but it may even be any relative path including sub-folders of sub1 and its templates.
 Given the `relocate` feature, you can even step out of the root path, which in general is the project/maven module the input is located in. This enables template designers to even address, e.g., maven modules located next to the module the input is coming from.
 
+=== Custom template-set groupIds for lookup
+
+You can specify template-sets so that it can provide custom template-sets and even restrict the teams to ask for default templates provided by CobiGen. There are `four` custom template-sets attributes:
+
+* `template-sets.groupIds`: Search for template-set artifacts by that configuration key
+configure multiple (comma separated) `groupIds`. By default, (public) CobiGen `groupId` will be used.
+
+* `template-sets.allow-snapshots=true`: Allow snapshots of template-sets to be offered for template-set development purposes. By default, no snapshots should be queried.
```

Comment:
Remove the value `true` from here. Just mention the key. The same for template-sets.disable-default-lookup.
I would prefer to have an example of how such a configuration file could look like after all the descriptions.

Reply:
Examples for each of these options should absolutely be given